src/contracts/core/RewardsCoordinator.sol

```diff
@@ -527,26 +527,4 @@ contract RewardsCoordinator is
         }
         revert InvalidRoot();
     }
-
-    /**
-     * @notice Getter function for the current EIP-712 domain separator for this contract.
-     *
-     * @dev The domain separator will change in the event of a fork that changes the ChainID.
-     * @dev By introducing a domain separator the DApp developers are guaranteed that there can be no signature collision.
-     * for more detailed information please read EIP-712.
-     */
-    function domainSeparator() public view returns (bytes32) {
```

Comment:
do we just not use EIP1271?

Reply:
Yeah, RC currently does not need signature utils.